dfastmi/batch/DFastUtils.py

```diff
@@ -132,3 +133,14 @@ def _get_km_bins(km_bin: Tuple[float, float, float], type_characteristic_chainag
     km = km_bin[0] + dx + numpy.arange(lb, ub) * km_step
 
     return km
+
+def get_progloc() -> str:
```

Comment:
Please use google doc strings as agreed

Reply:
is that something we can tackle codewide with black?

Reply:
After discussion with @MRVermeulenDeltares I think it might be best to adhere to the currently used style. As far I have seen up until now the most used style is Numpydoc:
https://numpydoc.readthedocs.io/en/latest/format.html

So lets be consistent with that.

Reply:
Will update but can you show me what I did wrong? I moved this code...